### DISCUSSION TITLE
how do i download

### DISCUSSION THREAD

**Discussion by BatuhanAkar** [Tyrrrz/YoutubeExplode#627] [Q&A]
how do i download audio and video files? I can't find.

**Comment by BatuhanAkar** [comment]
System.UnauthorizedAccessException: 'Access to the path "/HB_GnnhNz-8.mp4" is denied.'

i'm getting this error

**Reply by DaveCS1** [reply]
See Tyrrrz other project YoutubeDownloader. You can access the code there. 
https://github.com/Tyrrrz/YoutubeDownloader/tree/master/YoutubeDownloader.Core/Downloading
